src/containers/UserProfile/EditIntro.js

```diff
@@ -15,25 +15,27 @@ class EditInto extends Component {
       name: this.props.name,
       email: this.props.email
     };
-    this.handleUserInput = this.handleUserInput.bind(this);
     this.notificationDOMRef = React.createRef();
-    this.submitForm = this.submitForm.bind(this);
-    this.onCancel = this.onCancel.bind(this);
     this.modalRef = React.createRef();
     this.formRef = null;
   }
 
-  handleUserInput(e) {
+  handleUserInput = e => {
     const name = e.target.name;
     const value = e.target.value;
     this.setState({ [name]: value.trim() });
-  }
+  };
 
-  onCancel() {
+  onCancel = () => {
     this.formRef.reset();
-  }
+  };
 
-  submitForm(e) {
+  removeBackdrop = () => {
+    var parent = document.getElementsByClassName("modal-backdrop fade show");
+    parent[0].remove();
```

Comment:
@Humnaa rename `parent` to `element` and add check before accessing using `if(element && element.length)`

Reply:
done